docs/managing-dragonfly/using-http.md

```diff
@@ -0,0 +1,24 @@
+# HTTP
+
+Dragonfly supports HTTP for certain operations. By default, `HTTP` can be accessed on 
+the same port as the main `TCP` port. However, this can be turned off by the command
+`--primary_port_http_enabled=false`. Moreover, if `--admin_port` flag is set it also
+supports `HTTP` on admin port.
```

Comment:
I always have this question around HTTP: what's exactly is the relationship between HTTP and admin port?